chain/store/snapshot.go

```diff
@@ -35,15 +39,50 @@ func (cs *ChainStore) UnionStore() bstore.Blockstore {
 	return bstore.Union(cs.stateBlockstore, cs.chainBlockstore)
 }
 
-func (cs *ChainStore) Export(ctx context.Context, ts *types.TipSet, inclRecentRoots abi.ChainEpoch, skipOldMsgs bool, w io.Writer) error {
+func (cs *ChainStore) Export(ctx context.Context, ts *types.TipSet, inclRecentRoots abi.ChainEpoch, skipOldMsgs bool, f3Ds dtypes.F3DS, w io.Writer) error {
+	store, err := certstore.OpenStore(ctx, f3Ds)
+	if err != nil {
+		return xerrors.Errorf("failed to open certstore: %w", err)
+	}
+
+		log.Warnf("failed to export latest f3 snapshot: %v; continuing snapshot export without F3 data", err)
+	}
+
+	buffer := bytes.NewBuffer(nil)
+	metadata := SnapshotMetadata{
+		Version:       SnapshotVersion2,
+		HeadTipsetKey: ts.Cids(),
+	}
```

Comment:
I'm not sure if we are ok with exporting only v2 snapshots. I think it is ok but highlighting it.